lib/collective/collectors/trackjs.rb

```diff
@@ -0,0 +1,110 @@
+require 'pp'
+
+module Collective::Collectors
+  class TrackJS < Collective::Collector
+    requires :faraday
+    requires :faraday_middleware
+    requires :json
+
+    resolution '60s'
+
+    def initialize(lastSeenId = nil, options)
+      @lastSeenId = lastSeenId
+      super(options)
+    end
+
+    collect do
+      instrument_errors
+    end
+
+    private
+
+    def instrument_errors
+      count = 0
+      application = ""
+      paged("/#{customer_id}/v1/errors").each do |error|
+        application = error['application'] if count == 0
+        count += 1
+      end
+      instrument 'trackjs.url.errors', count, source: application, type: 'count'
```

Comment:
Weirdly when I originally wrote this I looked at our trackjs application settings in their web app and only saw an entry for r101-frontend, so I didn't do any application filtering in this version. I can add it though, since it'd be a good idea to have that sort of filtering ability build in by default anyway.